app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -35,7 +35,24 @@ public function index()
      */
     public function create()
     {
-        return view('campaign-ids.create');
+        $causes = [
+            'No Cause',
```

Comment:
I put `No Cause` as the first since this will be shown as default selected in the dropdown list - if it were any other campaign, I'd be worried people would skip this step and if so, campaigns would be incorrectly labeled as `Animals` for example and that cause would have many false labels. Would this cause the same problem though if they ignore this drop down and select `No Cause` (`No Cause` would have a lot of false labeling)? Not sure if I'm explaining this clearly...

Reply:
@chloealee makes sense! do we actually want to allow campaigns to not have causes? Should we make this field required and have a `-` placeholder?

Reply:
@katiecrane right now in Rogue admin (prod) at the bottom, there is a section that has `No Cause` with all the campaigns that don't have a cause! I guess in Ashes their `cause` would be `null` but I was thinking of just giving it a `No Cause` as the cause which is silly haha I think you're right that we should just have `-` as a placeholder but it has to be required (which it is now!) instead of `No Cause`. This is updated now to have a `Select the cause` placeholder! 

![screen shot 2018-11-13 at 4 07 45 pm](https://user-images.githubusercontent.com/9019452/48443183-4bd51d00-e75e-11e8-976e-72c5e6db8c34.png)

Reply:
@chloealee this looks awesome!!! Oooo interesting yeah I think maybe `null` for their cause would be best?

Reply:
@katiecrane right now, this is required so they must choose something from the dropdown otherwise a validation error pops up saying cause is required!